docs/DeveloperGuide.md

```diff
@@ -1,12 +1,220 @@
 # Developer Guide
 
 ## Acknowledgements
+Our project uses external libraries and services from:
+1. Land Transport Authority DataMall API Service ([link](https://datamall.lta.gov.sg/content/datamall/en.html)).
+2. Jackson JSON Parser ([link](https://fasterxml.github.io/jackson-core/javadoc/2.8/com/fasterxml/jackson/core/JsonParser.html))
+
+## Design 
+
+### Architecture Level
+
+### Component Level
+
+#### Model Component
+![Model Class Diagram](images/ModelClassDiagram.png)
```

Comment:
![image](https://user-images.githubusercontent.com/64647635/198495122-c5f3f69e-5a25-4c1c-bfc5-175e3aaa117f.png)
Should use the symbols taught in class like + or -